ui/js/constants.js

```diff
@@ -223,15 +223,14 @@ export const thFavicons = {
   unavailable: treeFavicon,
 };
 
-export const thRepoGroupOrder = {
-  development: 1,
-  'release-stabilization': 2,
-  'project repositories': 3,
-  'comm-repositories': 4,
-  'qa automation tests': 5,
-  try: 6,
-  taskcluster: 7,
-};
+export const thRepoGroupOrder = [
```

Comment:
I was able to simplify this object which simplified the logic in the ``ReposMenu`` component.  This DOES mean that if a new "Repository Group" is introduced that is not listed here, it won't show in the UI.  But I feel like that's ok.  I don't think that'll happen often, and we should be conscious of it's order in this list anyway.  

But if you disagree, I can change this.  :)

Reply:
Sounds fine to me; will make sure we remember to set an order for it